### DISCUSSION TITLE
Next release version is 0.4.0

### DISCUSSION THREAD

**Discussion by ManaSugi** [libseccomp-rs/libseccomp-rs#208] [Announcements]
@rusty-snake 

Next release version will be 0.4.0.
After libseccomp v2.6.0 is released, we might as well release libseccomp-rs v0.4.0 as soon as possible, so from now, we'll add new actions, architectures and functions and try to catch up with the latest libseccomp (main branch).

In the next release, rustc 1.63 will be MSRV due to `io_safety`.

- Deliverables:
  - libseccomp-rs: v0.4.0
  - libseccomp-sys: v0.3.0

**Comment by rusty-snake** [comment]
Note for me.
- [x] #179
- [ ] #166
- [x] #197
- [x] #180 (PoC)
